pyiron_base/project/generic.py

```diff
@@ -666,10 +666,7 @@ def create_function_job(*args, **kwargs):
             )
             job._automatically_rename_on_save_using_input = automatically_rename
             job.python_function = python_function
-            if not args and len(kwargs) == 0:
-                return job
-            else:
-                return job(*args, **kwargs)
```

Comment:
This code lead to a bug of python functions which do not require any arguments not being executed. It is now removed.